Exception/NavitiaException.php

```diff
@@ -6,17 +6,12 @@ class NavitiaException extends \Exception
 {
     /**
      * NavitiaException constructor.
+     *
+     * @param string $message
+     * @param \Exception $previous
      */
-    public function __construct($message, $code, $previous)
+    public function __construct($message = 'Navitia API call didn\'t end as expected', $previous = null)
     {
-        if (empty($message)) {
-            $message = 'Navitia API call didn\'t end as expected';
-        }
-
-        if (empty($code)) {
-            $code = 500;
-        }
-
-        parent::__construct($message, $code, $previous);
+        parent::__construct($message, 0, $previous);
```

Comment:
We don't use it for now, but we might wan't it later. Why don't you want to map navitia return status?
It would be nice for php users to catch an exception instead of looking the array response imho.